config/locales/it.yml

```
@@ -717,7 +717,7 @@ it:
   replyat: "Rispondi a "
   # reply at in views/email_reply/reply.text.erb
   emailreplyat: "%{author} ti ha risposto: "
-  emailcontinueto: "Continua la discussione su"
+  emailcontinueto: "Continua la discussione su "
   # reply at in views/email_reply/mention.text.erb
   emailmentionreplyat: "%{author} ti ha citato in un commento:"
   emailmentioncontinueto: "Continua la discussione su"
```

Penso che questa traduzione dia lo stesso problema

```suggestion
  emailmentioncontinueto: "Continua la discussione su "
```